src/GpxDocument.cpp

```diff
@@ -503,24 +505,24 @@ void OsmAnd::GpxDocument::writeExtensions(const QList< Ref<GpxExtension> > &exte
         xmlWriter.writeAttribute(attributeEntry.key(), attributeEntry.value());
 
     for (const auto& subextension : constOf(extensions))
-        writeExtension(subextension, xmlWriter);
+        writeExtension(subextension, xmlWriter, QStringLiteral("osmand:"));
 
     // </extensions>
     xmlWriter.writeEndElement();
 }
 
-void OsmAnd::GpxDocument::writeExtension(const std::shared_ptr<const GpxExtension>& extension, QXmlStreamWriter& xmlWriter)
+void OsmAnd::GpxDocument::writeExtension(const std::shared_ptr<const GpxExtension>& extension, QXmlStreamWriter& xmlWriter, const QString &namesp)
 {
     // <*>
-    xmlWriter.writeStartElement(extension->name);
+    xmlWriter.writeStartElement(namesp + extension->name);
```

Comment:
other method should be used !! https://doc.qt.io/qt-5/qxmlstreamwriter.html#writeStartElement